cristin-person/src/main/java/no/unit/nva/cristin/person/update/PersonPatchValidator.java

```diff
@@ -50,4 +63,19 @@ private static void validateReservedIfPresent(ObjectNode input) throws BadReques
             throw new BadRequestException(RESERVED_FIELD_CAN_ONLY_BE_SET_TO_TRUE);
         }
     }
+
+    private static void validateEmploymentsIfPresent(ObjectNode input) throws BadRequestException {
```

Comment:
I'd probably add the following just to make it more readable. Note, use the correct printf-message formatting in the EXCEPTION_WHEN_VALIDATING_EMPLOYMENTS, i.e. with {}.

```
    private static void validateEmploymentsIfPresent(ObjectNode input) throws BadRequestException {
        if (input.has(EMPLOYMENTS) && !input.get(EMPLOYMENTS).isNull()) {
                var employments = parseEmployments(input);
                for (Employment employment : employments) {
                    CreatePersonEmploymentValidator.validate(employment);
                }
        }
    }

    private static Employment[] parseEmployments(ObjectNode input) throws BadRequestException {
        try {
            return OBJECT_MAPPER.readValue(input.get(EMPLOYMENTS).toString(), Employment[].class);
        } catch (JsonProcessingException e) {
            logger.warn(EXCEPTION_WHEN_VALIDATING_EMPLOYMENTS, e.getMessage());
            throw new BadRequestException(COULD_NOT_PARSE_EMPLOYMENT_FIELD);
        }
    }
    
```